internal/tools/aws/cluster_management.go

```diff
@@ -203,11 +204,21 @@ func (a *Client) claimVpc(clusterResources ClusterResources, clusterID string, l
 	if err != nil {
 		return errors.Wrapf(err, "unable to update %s", VpcAvailableTagKey)
 	}
+
 	err = a.TagResource(clusterResources.VpcID, trimTagPrefix(VpcClusterIDTagKey), clusterID, logger)
 	if err != nil {
 		return errors.Wrapf(err, "unable to update %s", VpcClusterIDTagKey)
 	}
 
+	if owner == "" {
+		owner = "SRETeam"
```

Comment:
why not return here: https://github.com/mattermost/mattermost-cloud/pull/134/files#diff-e240f1301da8fcd8a09bc6dd63baf09bR242

Reply:
Agreed, that seems better.

Also, 0/5 on this, but we could lookup an env var instead of hardcoding SRETeam. That way we could change it between our AWS accounts if we want.

Reply:
Oh, haha, yeah it makes more sense to put the logic there doesn't it?

Reply:
I do like the env var suggestion @gabrieljackson but do you have a suggestion for the default value if the env var is not set?

Reply:
I combined these suggestions: the routine now first looks for an environment variable and uses that, if it is set. If not, it looks for git. If that still fails, it sets the string to the generic "SRETeam"